nuxt.config.js

```
@@ -43,6 +44,12 @@ export default {
           'https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap',
       },
     ],
+    script: [
+      {
+        src:
+          'https://maps.googleapis.com/maps/api/js?key=AIzaSyCPS6SZlor8qxfpul-dKyN6566XG2R5dFM&libraries=places ',
```

No hard coded urls or keys.

---

Read from env variables and fallback to hardcoded value.

---

Resolved

---

added GOOGLE_API_KEY, GOOGLE_API in .env file